.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,4 +7,8 @@
 3. [STEP_3]
 
 ### Review
-@[USER_NAME]
\ No newline at end of file
+@[USER_NAME]
+
+Make sure strings can be translated:
+
+- [ ] If there are new strings that have to be translated, I'm adding them to the client's `strings.xml` as a part of the integration PR.
```

Comment:
What do you think about having it in past tense like the WPAndroid case?

Like: "If there are new strings that have to be translated, I have added them to the client's `strings.xml` as a part of the integration PR."